druid-win-shell/src/window.rs

```diff
@@ -1,127 +1,22 @@
-// Copyright 2018 The xi-editor Authors.
```

Comment:
This project got forked from a Google opensource released project (xi-win), so I think it's most polite to keep the copyright boilerplate in place. You probably noticed I've been using a super lightweight approach on new crates.

If we wanted to rename (possibly as part of a repo move), I think we could do that.